code/modules/mob/living/silicon/silicon.dm

```diff
@@ -85,13 +85,13 @@
 	modularInterface.saved_identification = real_name || name
 	if(istype(src, /mob/living/silicon/robot))
 		modularInterface.saved_job = "Cyborg"
-		modularInterface.install_component(new /obj/item/computer_hardware/hard_drive/small/robot)
+		modularInterface.install_component(new /obj/item/computer_hardware/hard_drive/small/integrated/borg)
```

Comment:
Aside from missing //SKYRAT EDIT comments, what's the ACTUAL difference between /robot and /integrated/borg? Are there any other balance changes this could cause, such as being able to download apps when they shouldnt be able to?